workspaces/icecap-host/nix/host/token-ssh-key.priv

```diff
@@ -0,0 +1,7 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
+b3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW
+QyNTUxOQAAACDZSxpPUDv+GRRMGzRFDXkC1v2rXu/t9TTjxLY3pQ5O7QAAAIhRtU39UbVN
+/QAAAAtzc2gtZWQyNTUxOQAAACDZSxpPUDv+GRRMGzRFDXkC1v2rXu/t9TTjxLY3pQ5O7Q
+AAAEBDgjbre3BSU5GNtuOMOjHTEb9+q5urYyzYJVjz8O9Q/tlLGk9QO/4ZFEwbNEUNeQLW
+/ate7+31NOPEtjelDk7tAAAABHJvb3QB
+-----END OPENSSH PRIVATE KEY-----
```

Comment:
Is there an explanation anywhere (I don't think this file format gives you a place to do that) for why we are including a private key?

Reply:
This file already exits at `icecap/nix/host/token-ssh-key.priv`. I think it's there to allow logging into the Linux host (for development).

Reply:
Right, but do we have it documented anywhere? Placing a private key in a git repo is suspicious, so it should be noted somewhere why we are doing it.

Reply:
I will remove it